file.go

```diff
@@ -4,39 +4,62 @@ import (
 	"fmt"
 	"os"
 	"strconv"
+	"strings"
 	"sync/atomic"
 	"time"
 )
 
 // GenerateWarcFileName generate a WARC file name following recommendations
 // of the specs:
 // Prefix-Timestamp-Serial-Crawlhost.warc.gz
-func generateWarcFileName(prefix string, compression string, serial *atomic.Uint64) (fileName string) {
+func generateWarcFileName(prefix string, compression string, serial *atomic.Uint64) string {
```

Comment:
I will be THAT guy, but I think it should be generateWARCFilename (note the usage of Filename as a word, instead of File Name, and of course the WARC uppercasing)

Reply:
oke